### PR TITLE
Fix a broken link on the Deploying section.

### PR DESCRIPTION
**Pull request by jmccormick2001**
What type of PR is this?
/kind fix

What this PR does / why we need it:
This PR fixes a broken link on the deploying.html page with a working link relevant for this section/topic.

Which issue(s) this PR fixes:
https://github.com/kubernetes-csi/docs/issues/526

Special notes for your reviewer:

Does this PR introduce a user-facing change?
Fixes a broken link only, the user still sees the same text.

Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

### CODE REVIEW

**Comment by jmccormick2001** [comment]
ping

**Comment by jmccormick2001** [comment]
ping

File: book/src/deploying.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ on the host to see mounts created by the CSI driver container. See the example b
 
 
 ## Deploying
-Deploying a CSI driver onto Kubernetes is highlighted in detail in [_Recommended Mechanism for Deploying CSI Drivers on Kubernetes_](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/container-storage-interface.md#recommended-mechanism-for-deploying-csi-drivers-on-kubernetes). 
+Deploying a CSI driver onto Kubernetes is highlighted in detail in [_Recommended Mechanism for Deploying CSI Drivers on Kubernetes_](https://github.com/kubernetes/design-proposals-archive/blob/main/storage/container-storage-interface.md).
```
[the remark this answers is not in the document]
Comment (Kartik494) [reply book/src/deploying.md:102]: @jmccormick2001 Could you please push a commit with suggested changes to update link as mentioned here.

[the remark this answers is not in the document]
Comment (jmccormick2001) [reply book/src/deploying.md]: commit pushed

[the remark this answers is not in the document]
Comment (Kartik494) [reply book/src/deploying.md]: @jmccormick2001  thanks for the commit , could you please  confirmed if you have signed cla

[the remark this answers is not in the document]
Comment (jmccormick2001) [reply book/src/deploying.md]: thanks, I've amended the commit to include my github id, and also signed the CLA, not sure what else I need to do regarding the cla.

[the remark this answers is not in the document]
Comment (Kartik494) [reply book/src/deploying.md]: > thanks, I've amended the commit to include my github id, and also signed the CLA, not sure what else I need to do regarding the cla.

I am not so sure on this ideally that would work, and also this PR needs re base , would you mind to take a look.

**Pull request closed without merging**